html-forms/html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="email" id="email" name="email" />
+      <div>
+        <label for="why-is-coding-cool">Why is coding cool?</label>
+      </div>
+      <select id="why-is-coding-cool">
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+      <p>Favorite Language (choose one)</p>
+      <p>
+        <input type="radio" id="HTML" name="html" value="html" />
+        <label for="CSS">HTML</label>
+        <input type="radio" id="CSS" name="html" value="html" />
+        <label for="HTML">CSS</label>
```

Comment:
let's make sure the values for these attributes are matching properly (ie. the label for CSS should have text content of CSS, not HTML - the `value` attribute should also correspond.

The `name` attribute should match in all the radio elements, but maybe naming it something like `name="favoriteLanguage"` is more appropriate.